src/style.css

```diff
@@ -0,0 +1,62 @@
+body {
+  margin: 0;
+}
+
+.starstreak {
+  width: 96px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
```

Comment:
line 18 - should be `ul` rule I guess

Reply:
but it is in .stars class of <ul>. is it still wrong?

Reply:
Nazar means these resetting rules are logically related to the `ul` HTML tag and not your class